### ISSUE TITLE
Subaccounts flat rate split payments

### ISSUE THREAD

**Issue by thukuwanjiku** [wilburt/rave_flutter#17]
Looking a the [Subaccount class constructor](https://github.com/wilburt/rave_flutter/blob/18a2280a085ec22a0b266593ad74facc6ad4c9d3/lib/src/models/sub_account.dart#L9) it takes an **_id_** and **_transactionSplitRatio_**. This is convenient for percentage based sharing.
However I want to work with the [**_flat_subaccount_**](https://developer.flutterwave.com/docs/split-payment#section-passing-the-exact-amount-you-want-a-subaccount-to-receive) _transaction_charge_type_ and simply pass a static amount which I need shared to that particular subaccount. Just static amount and not based on ratio or percentage. For instance if total amount is KES 1,500, I want the subaccount to get KES 1, 050 which I have determined from my own calculations.

Please, how can I achieve this?

**Comment by wilburx9** [comment]
From the [documentation](https://developer.flutterwave.com/docs/split-payment#section-split-payment-parameters), I think this should be possible.

I will see about supporting it on the plugin.

**Comment by thukuwanjiku** [comment]
> From the [documentation](https://developer.flutterwave.com/docs/split-payment#section-split-payment-parameters), I think this should be possible.
> 
> I will see about supporting it on the plugin.

Great, thank you for considering it. I hope it'll be soon. Eagerly waiting..

**Comment by MarkSure** [comment]
Yeah from the documentation it's possible, but even after updating the Subaccount class I notice that the subaccount data is not being sent in the meta object, rather it sends "sdk" as metaname and "flutter" as metavalue instead of "subaccount_split" and the subaccount data respectively.
Instead of something like this:
  "meta": [
            {
                "id": 200247022,
                "metaname": "subaccount_split",
                "metavalue": "{\"subaccount\":\"RS_9B3C22C01E05ADA2CA1F9365437F2YT6\",\"transaction_charge\":100,\"transaction_split_ratio\":1,\"transaction_charge_type\":\"flat\"}",
                "createdAt": "2020-09-24T11:19:12.000Z",
                "updatedAt": "2020-09-24T11:19:12.000Z",
                "deletedAt": null,
                "getpaidTransactionId": 1558577
            }
        ]
You have something like this:
 "meta": [
            {
                "id": 200246633,
                "metaname": "sdk",
                "metavalue": "flutter",
                "createdAt": "2020-09-24T10:10:49.000Z",
                "updatedAt": "2020-09-24T10:10:49.000Z",
                "deletedAt": null,
                "getpaidTransactionId": 1558354
            }
        ]
